src/Nethermind/Nethermind.BeaconNode.Host/Development9001/appsettings.json

```diff
@@ -1,13 +1,36 @@
 {
+  "Environment": "Development",
+  "Kestrel": {
+    "Endpoints": {
+      "Http": {
+        "Url": "http://localhost:5001"
+      },
+      "HttpsDefaultCert": {
+        "Url": "https://localhost:8231"
+      }
+    }
+  },
+  "Peering": {
+    "Port": 9001
```

Comment:
I did not check it but from experience we should make sure that the port numbers are exactly the same as in other major clients to avoid confusion for users (I assume it is ok here)

Reply:
The default in Mothra is 9000; the main node has a blank parameter so uses the default, but it probably makes sense to put it in the default config so it is visible to users (rather than a hidden default).

The config above is actually for a second node (running on a different port, with a different data directory) for the simple peering demo.